tarantool/connection.py

```diff
@@ -1023,7 +1040,8 @@ def connect(self):
             if self.transport == SSL_TRANSPORT:
                 self.wrap_socket_ssl()
             self.handshake()
-            self.load_schema()
+            if self.schema.schema != {}:
```

Comment:
Relying on internals in such a way is not the best approach. And I think this one is also wrong -- `schema.schema` is `{}` is there are no spaces yet.

Reply:
[Done](https://github.com/tarantool/tarantool-python/pull/271#discussion_r1065453318)